---
title: Interpretable network-guided epistasis detection
date: '2020-09-01'
draft: true
publishDate: '2021-02-25T11:47:33.489605Z'
authors:
- Diane Duroux
- Héctor Climente-González
- Chloé-Agathe Azencott
- Kristel Van Steen
publication_types:
- '4'
abstract: Abstract Detecting epistatic interactions at the gene level is essential
  to understanding the biological mechanisms of complex diseases. Unfortunately, genome-wide
  interaction association studies (GWAIS) involve many statistical challenges that
  make such detection hard. We propose a multi-step protocol for epistasis detection
  along the edges of a gene-gene co-function network. Such an approach reduces the
  number of tests performed and provides interpretable interactions, while keeping
  type I error controlled. Yet, mapping gene-interactions into testable SNP-interaction
  hypotheses, as well as computing gene pair association scores from SNP pair ones,
  is not trivial. Here we compare three SNP-gene mappings (positional overlap, eQTL
  and proximity in 3D structure) and used the adaptive truncated product method to
  compute gene pair scores. This method is non-parametric, does not require a known
  null distribution, and is fast to compute. We apply multiple variants of this protocol
  to a GWAS inflammatory bowel disease (IBD) dataset. Different configurations produced
  different results, highlighting that various mechanisms are implicated in IBD, while
  at the same time, results overlapped with known disease biology. Importantly, the
  proposed pipeline also differs from a conventional approach were no network is used,
  showing the potential for additional discoveries when prior biological knowledge
  is incorporated into epistasis detection.
featured: false
publication: ''
url_pdf: http://biorxiv.org/lookup/doi/10.1101/2020.09.24.310136
doi: 10.1101/2020.09.24.310136
---

